python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S1536.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Function argument names should be unique",
```

Comment:
Why does the title mention "argument"? Shouldn't we fix that as well as the rule description?